client/app/components/Input/__tests__/InputTag.spec.jsx

```diff
@@ -3,24 +3,43 @@ import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { InputMocks } from 'mocks/InputMocks';
 
-const component = InputMocks.createInput(InputMocks.inputTagProps);
-const { checkboxes } = InputMocks.inputTagProps;
+const { inputTagProps, createInput } = InputMocks;
+const getComponent = (extraProps = {}) => createInput(inputTagProps, extraProps);
+// baseline for most tests
+const component = getComponent();
+const { checkboxes } = inputTagProps;
 
 describe('InputTag', () => {
-  it('type in value and select it', () => {
-    const [, { label: checkboxLabelTwo }] = checkboxes;
+  it('renders input correctly', () => {
     render(component);
 
     const combobox = screen.getByRole('combobox');
     const input = screen.getByRole('textbox');
+    const listbox = screen.getByRole('listbox');
 
     expect(combobox).toBeInTheDocument();
     expect(input).toBeInTheDocument();
+    expect(listbox).toBeInTheDocument();
+  });
+
+  it('toggles menu options correctly', () => {
+    render(component);
+    const input = screen.getByRole('textbox');
+    expect(screen.queryByRole('option')).not.toBeInTheDocument();
+    // toggle options by focusing on input
+    userEvent.click(input);
+    expect(screen.getAllByRole('option')).toHaveLength(2);
+  });
+
+  it('type in value and select it', () => {
+    const [, { label: labelTwo }] = checkboxes;
```

Comment:
Can this just be `[{ label: labelTwo }]`?

Reply:
For sure! I used the second checkbox label as the rendered mock component in these tests already selects the first checkbox value by default. But now that I look at it... the next test block overlaps with this one.

I'll try to separate those expectations, or maybe I can just remove this new test ("type in value and select it"), since the expectation is already essentially covered by the next block 👍